### ISSUE TITLE
webutil.user_exists fails using grep

### ISSUE THREAD

**Issue by morsik** [saltstack/salt#28166]
state:
```yaml
{% for user in salt['pillar.get']('icinga:users') %}
icinga_users_{{ user }}:
  webutil.user_exists:
    - name: {{ user }}
    - password: {{ salt['pillar.get']('icinga:users:' + user + ':password') }}
    - htpasswd_file: /etc/icinga/passwd
    - options: d
    - force: false
{%  endfor %}
```

pillar:
```yaml
# salt-call pillar.get icinga:users --output=yaml
local:
  admin:
    password: admin-pass
  developer:
    password: dev-pass
```

salt log:
```
[INFO    ] Executing command 'grep   admin /etc/icinga/passwd' in directory '/root'
[INFO    ] User already known
```

and `/etc/icinga/passwd ` file…
```
icingaadmin:bRH4xwLfPkKeE
```

This grep fails too much…

Anyway it's not possible to remove user nor clear users file somehow (without doing another state that clears file).

**Comment by jfindlay** [comment]
@morsik, thanks for the report.

**Comment by jfindlay** [comment]
@morsik, also, will you provide the version of salt you are using?  Thanks.

**Comment by morsik** [comment]
@jfindlay: I used latest Salt available in EPEL6: `2015.5.5-1.el6`
